subsys/ipc/ipc_service/backends/ipc_rpmsg_static_vrings.c

```diff
@@ -15,20 +15,21 @@
 #include <ipc/ipc_rpmsg.h>
 
 #include <drivers/mbox.h>
+#include <dt-bindings/ipc_service/static_vrings.h>
 
 #include "ipc_rpmsg_static_vrings.h"
 
 #define DT_DRV_COMPAT	zephyr_ipc_openamp_static_vrings
 
-#define WQ_PRIORITY		K_HIGHEST_APPLICATION_THREAD_PRIO
```

Comment:
Would I need to create an overlay that sets the same prio @ahasztag ? and where if yes ?

Reply:
also would you have a ci job I can trigger to verify bluetooth and 802154 work well together ?